.github/workflows/pull-request.yml

```
@@ -31,46 +33,46 @@ jobs:
+      
       - name: checking linting
         run: pnpm run lint:check
-
+      
       - name: check for unused files,exports and dependencies
         run: pnpm run knip:check
-
+      
       - name: Check if the source and target branches are different
         if: ${{ github.event.pull_request.base.ref == github.event.pull_request.head.ref }}
         run: |
```

actionlint (1.7.9)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)